near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -329,7 +354,7 @@ impl NonFungibleToken {
     ///   Typically the account will be the owner. If `None`, will not refund. This is useful for delaying refunding
     ///   until multiple tokens have been minted.
     ///
-    /// Returns the newly minted token
+    /// Returns the newly minted token and does not emit the mint event
```

Comment:
Why not?

Reply:
Because in you could want emit an event for multiple mints.